modules/compiler/vecz/source/transform/control_flow_conversion_pass.cpp

```diff
@@ -2944,8 +2974,12 @@ bool ControlFlowConversionState::Impl::blendInstructions() {
     Type *T = opDef->getType();
     const unsigned numPreds = std::distance(pred_begin(B), pred_end(B));
     Value *blend = nullptr;
-    PHINode *PHI =
-        PHINode::Create(T, numPreds, opDef->getName() + ".merge", &B->front());
+    PHINode *PHI = PHINode::Create(T, numPreds, opDef->getName() + ".merge");
+#if LLVM_VERSION_GREATER_EQUAL(18, 0)
+    PHI->insertBefore(B->begin());
```

Comment:
Do you think this could become a sensible `multi_llvm` helper? `insertAtBeginningOfBlock`? or something?

Reply:
Ah, because of `getFirstNonPHI`. Uh, if you add `insertAtIterator` would we not still need to use `#if LLVM*` at every use to distinguish between `begin()`/`front()`?